languages/elixir/exercises/concept/regular-expressions/.docs/introduction.md

```diff
@@ -0,0 +1,76 @@
+Regular expressions are a powerful tool for string in Elixir. Regular expressions in Elixir follow the **PCRE** specification (**P**erl **C**ompatible **R**egular **E**xpressions). String patterns representing the regular expression's meaning are first compiled then used for matching all or part of a string. The `=~/2` operator is useful to perform a boolean match on a string. A brief overview of regular expressions follows:
+
+To match _literal characters_, you can use the string as the pattern:
+
+```elixir
+regex = "foo-bar" |> Regex.compile!()
+"foo-bar" =~ regex
+# => true
+```
+
+Matching a range of characters using square brackets `[]` to denote a _character class_. This will many any one character to the characters in the class. You can also specify a range of characters like `a-z`, as long as the start and end represent a contiguous range of codepoints.
```

Comment:
> This will many any one character

This is missing some verb, but I can't figure out which one...

Reply:
Found it! `many = match`.